mungegithub/mungers/close-stale.go

```diff
@@ -109,6 +99,20 @@ func findLastHumanPullRequestUpdate(obj *github.MungeObject) (*time.Time, bool)
 	return lastHuman, true
 }
 
+func findLastHumanPullRequestUpdate(obj *github.MungeObject) (*time.Time, bool) {
+	pr, ok := obj.GetPR()
+	if !ok {
+		return nil, ok
+	}
+
+	comments, ok := obj.ListReviewComments()
+	if !ok {
+		return nil, ok
+	}
+
+	return findLastHumanPullRequestUpdateHelper(pr.CreatedAt, comments)
```

Comment:
I will create some helper functions at `/matchers/comment.go` handling the case for `github.PullRequestComment`. thanks for the patience in explaining it. i will remove changes on `close-stale.go`.